server.js

```diff
@@ -1,26 +1,47 @@
-const express = require('express')
-const app = express()
-const mongoose = require('mongoose')
-const port = 3000
+const express = require('express');
+const mongoose = require('mongoose');
+const app = express();
+const port = 3000;
+const route = require('./routes');
+const arr = require("./data.json");
+const { post, get, del } = require('./routes');
+
+app.use(express.json());
 
 const uri = 'mongodb+srv://Surya:sun@asapcluster.3osvgms.mongodb.net/?retryWrites=true&w=majority';
```

Comment:
Consider using environment variables to store sensitive information such as the MongoDB connection string. This would enhance the security of your application. [important]